couchdbkit/schema/base.py

```diff
@@ -130,8 +131,15 @@ def save(self, **params):
         self.validate()
         db = self.get_db()
 
+        for prop_name, prop in self.properties().iteritems():
+            if isinstance(prop, p.DateTimePropertyCouchDB) and prop.auto_now_add:
+                setattr(self, prop_name, datetime.utcnow())
+            elif isinstance(prop, p.DatePropertyCouchDB) and prop.auto_now_add:
+                setattr(self, prop_name, datetime.utcnow().date())
```

Comment:
This is kind of an edge case, but I noticed that this means auto_now_add will only work top-level on a doc. If it's inside a DocumentSchema, then it won't _error_ but it won't _work_ either. (Usually you want one or the other!)